src/main/java/ledger/user_interface/ui_controllers/window/MainPageController.java

```diff
@@ -179,6 +182,13 @@ private void openAutoTaggingEditor(ActionEvent actionEvent) {
         this.createModal(null, scene, "Automatic Tagging", true);
     }
 
+    private void openTagEditor(ActionEvent actionEvent) {
+        TagEditorTableWindow window = new TagEditorTableWindow();
+        Scene scene = new Scene(window);
+
+        this.createModal(null, scene, "Tag Editor", true);
```

Comment:
If you set the parent it locks out out main page like its supposed to. 
so this.getScene().getWindow() instead of null